### PR TITLE
 Rename from Amber Glow to Sprintnet and Update Network Address

### PR DESCRIPTION
**Pull request by adi-a11y**

### CODE REVIEW

File: packages/apps/src/Endpoints/Network.tsx [hunk 1]
```diff
@@ -58,10 +58,10 @@ function NetworkDisplay ({ apiUrl, className = '', setApiUrl, value: { isChild,
               {isRelay
                 ? t('Relay chain')
                 : paraId && paraId < 1000
-                  ? t('{{relay}} System', { replace: { relay } })
+                  ? t('{{relay}} ', { replace: { relay } })
                   : paraId && paraId < 2000
-                    ? t('{{relay}} Common', { replace: { relay } })
-                    : t('{{relay}} Parachain', { replace: { relay } })
+                    ? t('{{relay}} ', { replace: { relay } })
+                    : t('{{relay}} ', { replace: { relay } })
```
Comment (amarts) [inline packages/apps/src/Endpoints/Network.tsx:64]: nitpick: not sure what this for...

**Review by amarts — APPROVED** [review]

**Review by roynrishingha — CHANGES_REQUESTED** [review]

File: packages/apps/src/Endpoints/Network.tsx (hunk 1 above)
Comment (adi-a11y) [reply packages/apps/src/Endpoints/Network.tsx]: On the explorer, when you click on a chain, say `sparknet` it used to show `CORD Network parachain` as a description. now with latest commit, it shows `CORD Network testnet`

File: packages/apps-config/src/endpoints/liveNetworks.ts [hunk 2]
```diff
@@ -32,9 +32,9 @@ export const prodParasKusama: Omit<EndpointOption, 'teleport'>[] = [
     info: 'Amber glow',
     paraId: 2124,
     providers: {
-      'CORD Network': 'wss://staging.cord.network/',
+      'CORD Network': 'wss://sprintnet.cord.network/',
     },
-    text: 'Amber Glow (v2 staging)',
+    text: 'Sprintnet',
```
[the remark this answers is not in the document]
Comment (adi-a11y) [reply packages/apps-config/src/endpoints/liveNetworks.ts:37]: Ack. Thank you, Roy.

File: packages/apps/src/Endpoints/Network.tsx (hunk 1 above)
Comment (amarts) [reply packages/apps/src/Endpoints/Network.tsx]: Ah! Ok. Then this is a must have fix.

File: packages/apps-config/src/endpoints/liveNetworks.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (amarts) [reply packages/apps-config/src/endpoints/liveNetworks.ts:37]: Sure. Makes sense.

File: packages/apps/src/Endpoints/Network.tsx (hunk 1 above)
Comment (adi-a11y) [reply packages/apps/src/Endpoints/Network.tsx]: Yes and this is done

File: packages/apps-config/src/endpoints/liveNetworks.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (adi-a11y) [reply packages/apps-config/src/endpoints/liveNetworks.ts]: This is done. Thank you for the reviews.

**Review by amarts — APPROVED** [review]

**Pull request merged**